frost-p256/dkg.md

```diff
@@ -140,7 +140,7 @@ let mut key_packages = BTreeMap::new();
 
 // Keep track of each participant's public key package.
 // In practice, if there is a Coordinator, only they need to store the set.
-// If there is not, then all candidates must store their own sets.
+// If there is no, then all candidates must store their own sets.
```

Comment:
I think this was fine the way it was

Reply:
@mpguerra Hi! Replacing 'not' with 'no' would be correct if the meaning of the phrase is that something is absent, rather than a negation, but I will do it as you say.